clang/test/CXX/drs/cwg2xx.cpp

```diff
@@ -230,6 +230,37 @@ namespace cwg211 { // cwg211: 2.7
   };
 } // namespace cwg211
 
+namespace cwg212 { // cwg212: yes
```

Comment:
```suggestion
namespace cwg212 { // cwg212: 2.7
```
I checked on Compiler Explorer that we exhibit the expected behavior all the way down to 2.7 in both C++98 and C++11 modes

Reply:
The value "yes" historically meant "there was never a time when we implemented any other behavior", which is the case here. Has this changed to mean just "we don't know when we started supporting this" instead?

Reply:
Ah, yeah, I see that we've removed nearly all of them. I suppose it'd make sense to remove "yes" entirely as an option, given we're so close, and also given that there seems to have been more than one interpretation of what it means -- the latter is my fault, I should have called it "always" or something, or at least written down what I was thinking! :)

Reply:
> I suppose it'd make sense to remove "yes" entirely as an option, given we're so close

I discussed that with other maintainers, and it was met with some resistance. I guess DR status page for C being filled with "yes" didn't help.

(There is a dream for C DR tests to follow the example of C++ DR tests, including the script to automatically generate the status page.)